From b0e91f703ffdcc73f854a672ec16ebbf21293c9c Mon Sep 17 00:00:00 2001
From: Chase Maupin <chase.maupin@ti.com>
Date: Mon, 24 May 2010 12:51:49 -0500
Subject: [PATCH] Disable cursor override for X11

* Disable the cursor override option for using the matrix_gui
  program with X11.
---
 main.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/main.cpp b/main.cpp
index 978e327..1f55e94 100644
--- a/main.cpp
+++ b/main.cpp
@@ -43,7 +43,7 @@ int main(int argc, char * argv[])
 {
     QApplication matrix(argc, argv);
     QApplication::setStyle(new CustomStyle);
-    QApplication::setOverrideCursor(QCursor(Qt::BlankCursor));
+//    QApplication::setOverrideCursor(QCursor(Qt::BlankCursor));
 
     // Create a new window to display a web page.
     MainWindow * pWindow = new MainWindow(0, QString(argv[1]), true);
-- 
1.6.0.4

